### DISCUSSION TITLE
最佳测速时间段的探讨

### DISCUSSION THREAD

**Discussion by doublewinter0** [XIU2/CloudflareSpeedTest#275] [讨论]
对于自己所在区域和自己所使用的运营商，大家有没有在不同的时间段去测速，然后得出一些有趣的结论呢？

**Comment by GhostKeith** [comment]
目前没发现时间段对测试速度的影响，不过倒是发现了ip段对不同运营商有影响，部分ip段例如162.159.58.xxx联通的线路延迟就很低，有些ip段对移动的延迟很低，还有部分ip段对联通和移动的延迟都比较低，多次筛选这些ip段之后就能比较精准的选取延迟最低的ip。目前来看移动是比较香的有几个ip段移动延迟能在50以下，联通最低也就60-70的样子
